### PR TITLE
Ensure correct object is select when prompt is given

### PR DESCRIPTION
**Pull request by rondale-sc**
I was unable to get the test suite to run this test: received the following error:

It should work. :)

### CODE REVIEW

File: app/components/object-select/component.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  actions: {
+ actions: {
```
Comment (rwjblue) [inline app/components/object-select/component.js]: I believe this white space was correct (2 space indent) before, can you double check and update?

File: app/components/object-select/component.js [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
     update: function() {
+      let selected;
       const selectedEl = this.$('select')[0];
       const selectedIndex = selectedEl.selectedIndex;
       const items = Ember.A(this.get('items'));
-      const selected = items.objectAt(selectedIndex);
+
+      if (this.get('prompt')){
+        selected = items.objectAt(selectedIndex - 1);
```
Comment (rwjblue) [inline app/components/object-select/component.js]: Can you abstract this a little bit?

Something like:

    let promptOffset = this.get('prompt') ? 1 : 0;
    selected = items.objectAt(selectedIndex - promptOffset);

**Comment by rwjblue** [comment]
Thanks for fixing this (and adding a test)!

Left a few minor comments, but this looks great!

**Comment by rondale-sc** [comment]
Updated.

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**